python/lsst/obs/base/formatters/fitsExposure.py

```diff
@@ -528,7 +530,17 @@ class FitsExposureFormatter(FitsMaskedImageFormatter):
     can_read_from_uri = True
     ReaderClass = ExposureFitsReader
     # TODO: Remove MemFileManager from cache when DM-49640 is fixed.
-    _cached_fits: tuple[uuid.UUID | None, ExposureFitsReader | None, MemFileManager | None] = (
+    _cached_fits: tuple[
+        uuid.UUID | None,
+        ExposureFitsReader | None,
+        MemFileManager | None,
+        lsst.geom.Point2I | None,
+        lsst.geom.Extent2I | None,
+        lsst.geom.Box2I | None,
```

Comment:
I think it'd be simpler to always read and cache the `bbox` component when any size-based component is requested.  They're all derived from the `bbox`.

Reply:
This isn't the current version of the code is it? I do cache dimensions, xy0, and bbox.

Reply:
Yeah, I was looking commit-by-commit, but the point still holds: you don't need to cache all three, because
```
xy0 = bbox.getMin()
dimensions = bbox.getDimensions()
```